test/cloudtest/test_managed_cluster.py

```diff
@@ -28,6 +28,26 @@ def test_managed_cluster_sizing(mz: MaterializeApplication) -> None:
     )[0][0]
     assert cluster_id is not None
 
+    check = mz.environmentd.sql_query(
+        "SELECT availability_zones IS NULL FROM mz_clusters WHERE name = 'sized1'"
+    )[0][0]
+    assert check is not None
+    assert check == True
+
+    mz.environmentd.sql("ALTER CLUSTER sized1 SET (AVAILABILITY ZONES ('1', '2', '3'))")
+    check = mz.environmentd.sql_query(
+        "SELECT list_length(availability_zones) = 3 FROM mz_clusters WHERE name = 'sized1'"
+    )[0][0]
+    assert check is not None
+    assert check == True
+
+    try:
+        mz.environmentd.sql("ALTER CLUSTER sized1 SET (AVAILABILITY ZONES ('4'))")
+    except:
+        pass
+    else:
```

Comment:
Can you please assert here that the exact error you are expecting is indeed delivered. You can use a testdrive fragment if this will make things easier.

As written, the test will continue to pass on any error, for example, if the SQL syntax changes an the SQL statement becomes invalid altogether.

Reply:
will do